pkg/testworkflows/testworkflowprocessor/intermediate.go

```diff
@@ -97,6 +107,18 @@ func (s *intermediate) AppendPodConfig(cfg *testworkflowsv1.PodConfig) Intermedi
 	return s
 }
 
+func (s *intermediate) AppendPvcs(cfg map[string]corev1.PersistentVolumeClaimSpec) Intermediate {
+	for name, spec := range cfg {
+		s.Ps[name] = corev1.PersistentVolumeClaim{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: s.NextRef(),
```

Comment:
More likely:
```suggestion
				Name: fmt.Sprintf("{{resource.id}}-%s", ref.NextRef()),
```

As the `NextRef()` is unique in the context of the Execution, but not necessarily between multiple executions

Reply:
thank you, good catch, fixed!